src/main/java/uk/gov/hmcts/cmc/claimstore/services/DocumentsService.java

```diff
@@ -0,0 +1,72 @@
+package uk.gov.hmcts.cmc.claimstore.services;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.cmc.claimstore.documents.CountyCourtJudgmentPdfService;
+import uk.gov.hmcts.cmc.claimstore.documents.DefendantResponseCopyService;
+import uk.gov.hmcts.cmc.claimstore.documents.LegalSealedClaimPdfService;
+import uk.gov.hmcts.cmc.claimstore.events.DocumentManagementSealedClaimHandler;
+import uk.gov.hmcts.cmc.claimstore.events.solicitor.RepresentedClaimIssuedEvent;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+
+import java.util.Optional;
+
+@Service
+public class DocumentsService {
+    private final ClaimService claimService;
+    private final DefendantResponseCopyService defendantResponseCopyService;
+    private final LegalSealedClaimPdfService legalSealedClaimPdfService;
+    private final CountyCourtJudgmentPdfService countyCourtJudgmentPdfService;
+    private final DocumentManagementService documentManagementService;
+    private final DocumentManagementSealedClaimHandler documentManagementSealedClaimHandler;
+    private final boolean documentManagementFeatureEnabled;
+
+    @Autowired
+    public DocumentsService(
+        final ClaimService claimService,
+        final DefendantResponseCopyService defendantResponseCopyService,
+        final LegalSealedClaimPdfService legalSealedClaimPdfService,
+        final CountyCourtJudgmentPdfService countyCourtJudgmentPdfService,
+        final DocumentManagementService documentManagementService,
+        final DocumentManagementSealedClaimHandler documentManagementSealedClaimHandler,
+        @Value("${feature_toggles.document_management}") final boolean documentManagementFeatureEnabled
```

Comment:
This is not a proper way to do feature toggles. Application logic is polluted with toggling code. Please do the following:
1. create an interface named `DocumentsService`,
2. take the old code that was used before and create a `LegacyDocumentsService implements DocumentsService`,
3. take the new code and create a `DocumentsManagementDocumentsService implements DocumentsService`,
4. use Spring's [`@ConditionalOnProperty`](https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/autoconfigure/condition/ConditionalOnProperty.html) to instantiate either of the components based on the feature toggle value,
5. use the interface for service field type in the controller.